<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextSelectorMenu</class>
 <widget class="QDialog" name="TextSelectorMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>123</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>168</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>168</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#TextSelectorMenu{
background:transparent;
}

QLabel#line{
border: 1px solid #EDEDED;
}

QFrame#frame{
 background: #FFFFFF;
border-radius:4px;
}


QPushButton {
font-size:20px;
color:#555555;
font-family:&quot;lviconfont&quot;;
}

QPushButton:checked {
background: #F3F3F3;
border-radius: 4px;
}

QPushButton:hover {
background: #F3F3F3;
border-radius: 4px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="SizeSlideBar" name="slide" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="line">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>1</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>1</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="fontStypleWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>48</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="boldBtn">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>粗</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="italicBtn">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>斜</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="underLineBtn">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>28</width>
             <height>28</height>
            </size>
           </property>
           <property name="text">
            <string>底</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SizeSlideBar</class>
   <extends>QWidget</extends>
   <header>size_slide_bar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
